sympy/codegen/numpy_nodes.py

```diff
@@ -60,7 +60,7 @@ def _eval_evalf(self, *args, **kwargs):
         return self.rewrite(log).evalf(*args, **kwargs)
 
     def _eval_simplify(self, *args, **kwargs):
-        a, b = map(lambda x: x.simplify(**kwargs), self.args)
+        a, b = (x.simplify(**kwargs) for x in self.args)
```

Comment:
Side comment:

I wish we wouldn't have two element loops like this. This line would fail if there are not two args so just write the signature like that:
```python
def _eval_simplify(self, a, b, **kwargs):
    a = a.simplify(**kwargs)
    b = b.simplify(**kwargs)
```
The repetition is fine: it is not worth complicating things just to reduce a single simple duplicate call.

Reply:
@oscarbenjamin Actually, make this change apparently broke tests. https://github.com/sympy/sympy/actions/runs/4214044376/jobs/7314304910 reverting for now

Reply:
How can that break anything? Is something catching an exception from this somehow?

Reply:
Ah, @oscarbenjamin `self.args != args`

Reply:
Ah okay.